app/package.json

```diff
@@ -6,8 +6,8 @@
     "@emotion/react": "^11.11.0",
     "@emotion/styled": "^11.11.0",
     "@fuel-ui/css": "^0.23",
-    "@fuels/connectors": "^0.2.2",
-    "@fuels/react": "^0.18.1",
+    "@fuels/connectors": "0.4.0",
```

Comment:
`fuels/connectors` v0.5.0 is released and it has a important fix to the ethereum wallet connectors, maybe worth upgrading ?

Reply:
Done